docs/docs/code/nodejs/working-with-files/README.md

```diff
@@ -20,9 +20,13 @@ Use the [`fs` module](https://nodejs.org/api/fs.html) to write data to `/tmp`:
 import fs from "fs"
 import { file } from 'tmp-promise'
 
-const { path, cleanup } = await file();
-await fs.promises.appendFile(path, Buffer.from("hello, world"))
-await cleanup();
```

Comment:
These modified examples look great. I didn't see where you linked to this file in `config.js` so I think it's still not accessible.